tests/gce_disk_integration_test.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Integration tests for GCE scratch disks."""
+
+import os
+import unittest
+
+from perfkitbenchmarker import pkb
+from perfkitbenchmarker import test_util
+
+MOUNT_POINT = '/scratch'
+
+
+@unittest.skipUnless('PERFKIT_INTEGRATION' in os.environ,
+                     'PERFKIT_INTEGRATION not in environment')
+class GcpScratchDiskIntegrationTest(unittest.TestCase):
```

Comment:
Please add a docstring explaining what these tests are doing. Since they involve actually interacting with a machine and may require a certain setup prior to running the test, there should be some resource for a user who wants to run all tests but doesn't know what these ones should do but then sees them failing.